examples/model_selection/plot_precision_recall.py

```diff
@@ -7,55 +7,53 @@
 
 Precision-Recall is a useful measure of success of prediction when the
 classes are very imbalanced. In information retrieval, precision is a
-measure of result relevancy, while recall is a measure of how many truly
-relevant results are returned.
-
-The precision-recall curve shows the tradeoff between precision and
-recall for different threshold. A high area under the curve represents
-both high recall and high precision, where high precision relates to a
-low false positive rate, and high recall relates to a low false negative
-rate. High scores for both show that the classifier is returning accurate
-results (high precision), as well as returning a majority of all positive
-results (high recall).
-
-A system with high recall but low precision returns many results, but most of
-its predicted labels are incorrect when compared to the training labels. A
-system with high precision but low recall is just the opposite, returning very
-few results, but most of its predicted labels are correct when compared to the
-training labels. An ideal system with high precision and high recall will
-return many results, with all results labeled correctly.
+measure of result relevancy, while recall is a measure of how many of the
+relevant results are returned. 'Relevancy' here refers to items that are
+postively labeled, true positives and false negatives.
```

Comment:
I think we can avoid introducing the word "relevancy" and more directly state:

```suggestion
measure of the fraction of relevant items among actually returned items while recall
is a measure of the fraction of items that were returned among all items that should
have been returned.
```

Reply:
Thanks!